ds/outputs/data/processed/_articles/airborne_count.json

```diff
@@ -0,0 +1 @@
+{"api_doc_url": "DAPS", "api_type": "MySQL", "data_date": "add_date", "description": "Number of preprints in a Covid-19 research cluster", "endpoint_url": "DAPS", "framework_group": "publications", "is_experimental": true, "order": ["year", "region_type", "region_year_spec", "region_id", "region_level", "value.id"], "region_types": ["NUTS"], "schema": {"region_id": "string", "region_level": "int", "region_type": "string", "region_year_spec": "int", "value": {"data_type": "int", "format": ".1f", "id": "airborne_count", "label": "Number participations in Airborne preprints"}, "year": {"data_type": "int", "label": "Year"}}, "source_name": "Nesta", "source_url": "https://github.com/nestauk/eurito_indicators", "subtitle": "Number participations in Airborne preprints by institutions in the location (research clusters identified through a topic model of article abstracts)", "title": "Number of participations in Airborne preprints", "date": "2021-10-19"}
```

Comment:
(2) It would be useful if you could indent the schemas to facilitate reviewing
(3) some fields have incorrect values, e.g. `"data_date": "add_date",` or `"api_doc_url": "DAPS"`

Reply:
What do you mean by indent the schema? 
Re: `api_doc_url` there isn't a URL with documentation so unsure about what to put there.

Reply:
> What do you mean by indent the schema?

I mean going from 

```
{"api_doc_url": "DAPS", "api_type": "MySQL", ...}
```

to

```
{
  "api_doc_url": "DAPS",
  "api_type": "MySQL",
  ...
}
```
so that we can refer to specific lines in the schema

Reply:
> Re: api_doc_url there isn't a URL with documentation so unsure about what to put there.

It seems like `api_doc_url` is optional so in that case I'd skip it